### PR TITLE
Disambiguation of WMS Axis Ordering differences

### PR DESCRIPTION
**Pull request by dgardim**
<Include a few sentences describing the overall goals for this Pull Request>

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates

Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.

### CODE REVIEW

File: doc/en/user/source/services/wms/basics.rst [hunk 1]
```diff
@@ -38,18 +38,15 @@ The major differences between versions 1.1.1 and 1.3.0 are:
...
-The WMS 1.3.0 specification mandates that the axis ordering for geographic 
-coordinate systems defined in the EPSG database be *latitude/longitude*, or *y/x*. 
-This is contrary to the fact that most spatial data is usually in 
-*longitude/latitude*, or *x/y*. 
-This requires that the coordinate order in the ``BBOX`` parameter
-be reversed for ``SRS`` values which are geographic coordinate systems.
+The WMS 1.3.0 specification mandates using the axis ordering as defined in the EPSG database. For instance, for EPSG:4326 the axis ordering is *latitude/longitude*, or *north/east*. This is contrary to the fact that most spatial data is usually in *longitude/latitude*, or *east/north*. 
 
-For example, consider the WMS 1.1 request using the WGS84 SRS (EPSG:4326):: 
+The WMS 1.1 always uses *east/north* axis ordering. So when upgrading from WMS 1.1, this requires the coordinate order in the ``BBOX`` parameter to be reversed for ``SRS`` with *north/east* axis ordering.
```
Comment (aaime) [inline doc/en/user/source/services/wms/basics.rst]: Wrong, it depends on the CRS. Many CRSs in the EPSG database have east/north orientation, almost (but not all) projected ones in particular. So for example, you would not switch ordinates if the BBOX is expressed in EPSG:3857 (web mercator) or EPSG:32632 (UTM 32N)

Comment (dgardim) [reply doc/en/user/source/services/wms/basics.rst]: Agree, it is not always required, that's why I wrote "for ``SRS`` with *north/east* axis ordering".
Maybe I could write it another way to be clearer about that.
For instance:
"The WMS 1.1 always uses *east/north* axis ordering. So when upgrading from WMS 1.1, if the CRS defines *north/east* axis ordering (e.g. EPSG:4326), the coordinate order in the ``BBOX`` parameter must be reversed."

Comment (aaime) [reply doc/en/user/source/services/wms/basics.rst]: Yes, it's clearer to me that way, thank you!

Comment (dgardim) [reply doc/en/user/source/services/wms/basics.rst]: tks

**Comment by aaime** [comment]
Looking good, merged, thanks!

**Pull request merged**